app/build.gradle

```diff
@@ -1,10 +1,18 @@
 apply plugin: 'com.android.application'
-
+// NOTE: you MUST disable configureOnDemand for your compiler.
+// To do this in Android Studio: https://stackoverflow.com/a/49994951/4082582
```

Comment:
Doesn't Android Studio yell at you about this by default? Does it need a note?

Reply:
Not really.
> Configuration on demand is not supported by the current version of the Android Gradle plugin since you are using Gradle version 4.6 or above. Suggestion: disable configuration on demand by setting org.gradle.configureondemand=false in your gradle.properties file or use a Gradle version less than 4.6.

This solution doesn't work, because Android Studio passes the argument as a command line flag, overriding the user's setting. It has to be manually turned off in Android Studio.